tmc-prototype/DishMaster/DishMaster/DishMaster.py

```diff
@@ -85,27 +87,29 @@ def increment_position(self,argin):
             temp1 = abs(self._elevation_difference)
         else:
             temp1 = temp
-            pass
 
         if temp == temp1:
             temp = temp + 1
-        else:
-            pass
 
-        for position in range(0,temp):
-            self.set_status("Dish is pointing towards the desired coordinates.")
-            self.devlogmsg("Dish is pointing towards the desired coordinates.", 4)
+        for position in range(0, temp):
+            self.set_status(CONST.STR_DISH_POINT_INPROG)
+            self.devlogmsg(CONST.STR_DISH_POINT_INPROG, 4)
             self._pointing_state = 1
             time.sleep(2)
-            if (self._achieved_pointing[1] == self._desired_pointing[1]) and (self._achieved_pointing[2] == self._desired_pointing[2]):
+            if (self._achieved_pointing[1] == self._desired_pointing[1])\
+                and (self._achieved_pointing[2] == self._desired_pointing[2]):
                 self._pointing_state = 0
-                self.set_status("Dish has pointed towards the desired coordinates.")
-                self.devlogmsg("Dish has pointed towards the desired coordinates.", 4)
-                pass
+                self.set_status(CONST.STR_DISH_POINT_INPROG)
+                self.devlogmsg(CONST.STR_DISH_POINT_INPROG, 4)
             else:
                 self._achieved_pointing[argin[0]] = self._achieved_pointing[argin[0]] + 1
 
-    def decrement_position(self,argin):
+    def decrement_position(self, argin):
+        """
+        This method gradually the decrements the Dish to the desired coordinates
```

Comment:
..gradually decrements... remove 'the'.

Reply:
Updated code documentation for CentralNode, SubarrayNode, DishLeafNode and DishMaster. Commit ID is ef9a34b.